scylla-macros/src/serialize/row.rs

```diff
@@ -205,40 +235,71 @@ impl Generator for ColumnSortingGenerator<'_> {
                 .push(syn::GenericParam::Lifetime(partial_lt.clone()));
         }
 
-        let (partial_impl_generics, partial_ty_generics, partial_where_clause) =
-            partial_generics.split_for_impl();
+        let flattened: Vec<_> = self.ctx.fields.iter().filter(|f| f.attrs.flatten).collect();
+        let flattened_fields: Vec<_> = flattened.iter().map(|f| &f.ident).collect();
+        let flattened_tys: Vec<_> = flattened.iter().map(|f| &f.ty).collect();
 
```

Comment:
Nit: Why use `tys`?
For a singular we can't use `type` because it is reserved in Rust (which is imo a mistake). We usually use `typ`, not sure why `ty` is used here.
For plural I see no reason to use `tys` instead of `types`.

Reply:
Yeah let's do that. It will be more consistent with the rest of this project.

Reply:
There are untouched spots right now where `ty` is used instead of `typ`. Would you want me to address those too?

Reply:
You can if you have time, but it is of course not required to merge this PR.

Reply:
I did it for the easy ones but in `DeserializeRow` and `DeserializeValue`  they need to stay as `ty` since the `darling` crate depends on those names.